CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 * [ENHANCEMENT] Added the ability to have a per tenant max search duration. [#1421](https://github.com/grafana/tempo/pull/1421) (@joe-elliott)
 * [ENHANCEMENT] Azure Backend: Add support for authentication with Managed Identities. [#1457](https://github.com/grafana/tempo/pull/1457) (@joe-elliott)
 * [ENHANCEMENT] metrics-generator: expose max_active_series as a metric [#1471](https://github.com/grafana/tempo/pull/1471) (@kvrhdn)
+* [ENHANCEMENT] Add s3 config option `insecure_skip_verify` [#1470](https://github.com/grafana/tempo/pull/1470) (@zalegrala)
```

Comment:
Consider changing Add to Added to it matches other entries in the changelog. If s3 is Amazon S3, then the S should probably be capitalized.
```suggestion
* [ENHANCEMENT] Added S3 configuration option `insecure_skip_verify` [#1470](https://github.com/grafana/tempo/pull/1470) (@zalegrala)
```

Reply:
Thanks for the review.  I've updated, though I do see a split in the changelog.